README.md

```diff
@@ -175,6 +175,11 @@ This feature allows to offload the OVS data-plane to the NIC while maintaining O
 can be clustered. The databases are OVN_Northbound and OVN_Southbound. This document explains how to 
 cluster them and start various daemons for the ovn-kubernetes integration.
 
+[External IP and LoadBalancer Ingress in Shared Gateway mode](./docs/external-ip-and-loadbalancer-ingress-in-shared-gateway-mode.md)
```

Comment:
why are we specifying "in Shared Gateway Mode"?

Reply:
done